### PR TITLE
Redirect audio tutorials to torchaudio

### PR DESCRIPTION
**Pull request by mthrok**

### CODE REVIEW

**Comment by svekars** [comment]
I think we should add all of the tutorials that we are redirecting to the this list: https://github.com/pytorch/tutorials/blob/main/.jenkins/validate_tutorials_built.py#L11 with a comment that these files are redirects now.

**Comment by carljparker** [comment]
> I think we should add all of the tutorials that we are redirecting to the this list: https://github.com/pytorch/tutorials/blob/main/.jenkins/validate_tutorials_built.py#L11 with a comment that these files are redirects now.

If we add the redirected tutorials to that file, will the all the CircleCI checks pass? Is that the reason that some of them are not passing?

**Comment by svekars** [comment]
yes, the check fails as expected. Adding to the NOT RUN list will fix it.

File: beginner_source/audio_data_augmentation_tutorial.rst [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+"""
+Audio Data Augmentation
+=======================
+
+This tutorial has been moved to https://pytorch.org/audio/stable/tutorials/audio_data_augmentation_tutorial.html
+
+It will redirect in 3 seconds.
+.. raw::html
+   <meta http-equiv="Refresh" content="3; url='https://pytorch.org/audio/stable/tutorials/audio_data_augmentation_tutorial.html'" />
+"""
```
Comment (svekars) [inline beginner_source/audio_data_augmentation_tutorial.rst:11]:
```suggestion
```

File: beginner_source/audio_data_augmentation_tutorial.rst [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
```
Comment (svekars) [inline beginner_source/audio_data_augmentation_tutorial.rst:1]:
```suggestion
```

File: beginner_source/audio_data_augmentation_tutorial.rst [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+"""
```
Comment (svekars) [inline beginner_source/audio_data_augmentation_tutorial.rst:2]:
```suggestion
```

File: beginner_source/audio_data_augmentation_tutorial.rst [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+Audio Data Augmentation
+=======================
+
+This tutorial has been moved to https://pytorch.org/audio/stable/tutorials/audio_data_augmentation_tutorial.html
+
+It will redirect in 3 seconds.
+.. raw::html
```
Comment (svekars) [inline beginner_source/audio_data_augmentation_tutorial.rst:9]:
```suggestion
.. raw:: html
```

**Review by svekars — COMMENTED** [review]
A couple of things to fix to ensure the html is not broken. I commented on one but applies to all files.

**Review by svekars — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**